src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_stacks/adf_default_pipeline.py

```diff
@@ -44,6 +44,10 @@ def generate_adf_default_pipeline(scope: core.Stack, stack_input):
     if "github" in _source_name:
         adf_github.GitHub.create_webhook_when_required(scope, _pipeline.cfn, stack_input["input"])
 
+    pipeline_triggers = stack_input["input"].get("triggers", {}).get("triggered_by", None)
+    if pipeline_triggers:
+        for k, v in pipeline_triggers.items():
+            _pipeline.add_pipeline_trigger((k,v))
```

Comment:
Can you use named arguments for this function call? With the new trigger type and params that would make it easier to understand what is happening here.